README.md

```diff
@@ -15,7 +15,7 @@ The React SDK is built on a new and modernized UI technology stack (the Constell
 
 ## Pega Infinity Server and Constellation-enabled Application
 
-This **8.7 version** of the React SDK assumes that you have access to a Pega Infinity server (**8.7.0+ GA**) running an application that is configured to run using the Constellation UI service.
+This **8.8 and 8.7 version** of the React SDK assumes that you have access to a Pega Infinity server (**8.8.0+ GA or 8.7.0+ GA**) running an application that is configured to run using the Constellation UI service.
```

Comment:
We can just say, "This version of the React SDK....". Specifying a release number after "This" could be mistaken to mean that there are other versions of SDK for the same release.

Reply:
Thanks. Just updated that simplification.